modules/37_industry/subsectors/declarations.gms

```diff
@@ -31,7 +31,7 @@ Parameters
   p37_mat2ue(all_enty,all_in)                                                  "Contribution of process output to ue in CES tree; Trivial if just one material per UE, as in steel [Gt/Gt]"
   p37_captureRate(all_te)                                                      "Capture rate of CCS technology"
   p37_selfCaptureRate(all_te)                                                  "Share of emissions from fossil fuels used for a CCS process which are captured by the CCS process itself"
-  p37_priceMat(all_enty)                                                       "Prices of external material input [2005$/kg = trn2005$/Gt]"
+  p37_priceMat(all_enty)                                                       "Prices of external material input [US$/kg] = [trn$US/Gt]"
```

Comment:
Why not US$2017? I would prefer to always have the reference years in the unit.

Reply:
this is a pure GAMS internal parameter and the reference year depends only on the unit of the input data